packages/stacking/src/index.ts

```diff
@@ -103,6 +105,49 @@ export interface LockStxOptions {
   burnBlockHeight: number;
 }
 
+/**
+ * Delegate stx options
+ *
+ * @param  {BigNum} amountMicroStx - number of microstacks to delegate
+ * @param  {String} delegateTo - the STX address of the delegatee
+ * @param  {number} untilBurnBlockHeight - the burnchain block height after which delegation is revoked
+ * @param  {String} poxAddress - the reward Bitcoin address of the delegator
+ * @param  {String} privateKey - private key to sign transaction
```

Comment:
FYI - you don't need to include the types in brackets, because TSDoc will generate them automatically. It leads to errors, too. For example, the type of `amountMicroStx` is `BN`, not `BigNum`. The type of `poxAddress` is `string | undefined`. I recommend just removing the explicit types.

Reply:
done as well